src/applications/disability-benefits/781/config/form.js

```diff
@@ -10,7 +10,11 @@ import ConfirmationPage from '../containers/ConfirmationPage';
 
 // const { } = fullSchema.definitions;
 
-import { introductionText } from '../helpers';
+import {
+  introductionText,
+  // incidentIntroduction781,
```

Comment:
Did these get replaced?

Reply:
@imeecuison Can we remove these, if they aren't used?

Reply:
Removed!